common/common.h

```diff
@@ -429,9 +429,11 @@ struct common_params {
     int32_t n_cache_reuse     = 0;            // min chunk size to reuse from the cache via KV shifting
     int32_t n_ctx_checkpoints = 8;            // max number of context checkpoints per slot
     int32_t cache_ram_mib     = 8192;         // -1 = no limit, 0 - disable, 1 = 1 MiB, etc.
+    int32_t local_media_max_size_mb = 15; // max size of loaded local media files
```

Comment:
vertically align the comments

```suggestion
    int32_t cache_ram_mib     = 8192;         // -1 = no limit, 0 - disable, 1 = 1 MiB, etc.
    int32_t local_media_max_size_mb = 15;     // max size of loaded local media files
```